docs/admin/administration.md

```diff
@@ -63,38 +63,108 @@ severity, message, description, and cause.
+                        to an empty string then all debug streams will be
+                        enabled
+      -s, --subsystems= This input string is equivalent to the use of the
+                        DD_SUBSYS environment variable and can be set to enable
+                        logging for specific subsystems or facilities. The
+                        expected syntax is a comma separated list of facility
+                        identifiers. Accepted DAOS facilities are
+                        common,tree,vos,client,server,rdb,pool,container,object-
+                        ,placement,rebuild,tier,mgmt,bio,tests, Common
+                        facilities (GURT) are MISC,MEM and CaRT facilities
+                        RPC,BULK,CORPC,GRP,LM,HG,ST,IV If not set, subsystems
+                        to enable will be read from server config file and if
+                        set to an empty string then logging all subsystems will
+                        be enabled
+```
+
+If an arg is not passed, then that logging parameter for each engine process is reset to the
+values set in the server config file that was used when starting `daos_server`.
+- `--masks` will be reset to the value of the engine config `log_mask` parameter.
+- `--streams` will be reset to the `env_vars` `DD_MASK` environment variable value or to an empty
+string if not set.
+- `--subsystems` will be reset to the `env_vars` `DD_SUBSYS` environment variable value or to an
+empty string if not set.
 
 Example usage:
 ```
-dmg server set-logmasks -m ERR,mgmt=DEBUG
+dmg server set-logmasks -m DEBUG,MEM=ERR -d mgmt,md -s server,mgmt,bio,common
 ```
 
-The masks input string should look like PREFIX1=LEVEL1,PREFIX2=LEVEL2,... where the syntax is
-identical to what is expected by the 'D_LOG_MASK' environment variable.
-If the 'PREFIX=' part is omitted, then the level applies to all defined facilities (e.g., a value
-of 'WARN' sets everything to WARN).
+This example would be a runtime equivalent to setting the following in the server config file:
+```
+...
+engines:
+- log_mask: DEBUG,MEM=ERR
+  env_vars:
+  - DD_SUBSYS=server,mgmt,bio,common
+  - DD_MASK=mgmt,md
+...
+```
 
-Supported priority levels for engine logging are FATAL, CRIT, ERR, WARN, NOTE, INFO, DEBUG.
+If the above server config file was used to start an engine process, running `dmg server
+set-logmasks` without parameters would reset logging to config values and would be equivalent to the
+example given above.
 
-For usage of streams ('DD_MASK') and subsystems ('DD_SUBSYS') parameters, logging is described in
-detail in the
-[`Debugging System`](https://docs.daos.io/v2.6/admin/troubleshooting/#debugging-system)
-section.
+For more information on the usage of masks (`D_LOG_MASK`), streams (`DD_MASK`) and subsystems
+(`DD_SUBSYS`) parameters refer to the
+[`Debugging System`](https://docs.daos.io/v2.6/admin/troubleshooting/#debugging-system) section.
```

Comment:
Existing, but
1. Should Debugging System be formatted like this? Maybe Just
```
[Debugging System]
```
2. Can these links not be dynamic? I.e. they are hardcoded to `docs.daos.io/v2.6`

Reply:
yes to both, we should probably apply both points across the admin guide